changelog/4712.feature.rst

```diff
@@ -0,0 +1 @@
+Add map source for the EUI instrument on Solar Orbiter
```

Comment:
```suggestion
`sunpy.map.GenericMap` now supports the EUI instrument on Solar Orbiter.
```

Reply:
Well, that doesn't quite make sense.  The base class doesn't support the new data in particular.  Point to the `Map` factory, to the new subclass, or to both.